### ISSUE TITLE
KeyboardHelpSection uses undefined option `a11yContentTagName`.

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/scenery-phet#760]
I ran into problems when attempting to convert `KeyboardHelpSection` to TS.  Option `a11yContentTagName` is not used in `KeyboardHelpSection` or its superclass `ReadingBlock( VBox, 0 )`.

In KeyboardHelpSection constructor:

```js
    options = merge( {
...
      // pdom - tag name for the entire content, usually content is a list of items
      a11yContentTagName: 'ul'
    }, options );
```

When I converted this to `optionized`, a TS error occurred for `a11yContentTagName`.  It's not defined in the parent options `ReadingBlockOptions & VBoxOption`. 

Searching the entire PhET code base, I found additional usages of `a11yContentTagName` (all in .js code, which may also be errors) but no definition in a base class. The complete set of usages is:

```
// BASEKeyboardHelpContent.js
      a11yContentTagName: null // just a paragraph for this section, no list

// CoulombsLawKeyboardHelpContent.js
      merge( {}, options, { a11yContentTagName: null } )

// ComboBoxKeyboardHelpSection.js
      a11yContentTagName: 'ol', // ordered list

// KeyboardHelpSection.js
      a11yContentTagName: 'ul'
      a11yContentTagName: null
```

So... Is `a11yContentTagName` vestigial? renamed? missing from superclass options? ...

**Comment by jessegreenberg** [comment]
In my working copy `a11yContentTagName` is forwarded to an `iconVBox` in the `KeyboardHelpSection` constructor. Thanks for converting and let me know if you would like me to continue with any of these files.

**Comment by pixelzoom** [comment]
Yes, I see that. But VBoxOptions has no `a11yContentTagName`, see below.  Where is this option defined, where is it documented, where is it used?

<img width="1536" alt="screenshot_1795" src="https://user-images.githubusercontent.com/3046552/180499676-6200d820-8d65-4a43-acd1-c08ebadfcd07.png">

**Comment by pixelzoom** [comment]
Oh, I see:

```
      tagName: options.a11yContentTagName
```

So `a11yContentTagName` should be included in `SelfOptions` for KeyboardHelpSection.
